src/navigation/AuthStack.js

```diff
@@ -92,17 +92,20 @@ const AuthStack = () => {
     // want to display anything then you can use a LOADER here
     return null;
   }
-
-  if (isFirstLaunch === true) {
+  if (isFirstLaunch) {
     routeName = 'Onboarding';
   } else {
     routeName = 'Login';
   }
 
   return (
-    <Stack.Navigator initialRouteName={routeName} onStateChange={onNavigationStateChange(user)}>
-      <Stack.Screen name="Onboarding" component={OnboardingScreen} options={header} />
-
+    <Stack.Navigator
+      initialRouteName={routeName}
+      screenOptions={{
+        header: () => null,
+      }}
+      onStateChange={state => onNavigationStateChange(state)}>
+      <Stack.Screen name="Onboarding" component={OnboardingScreen} />
```

Comment:
This reverses a vital change that allows for a significant performance boost. Please try merging with main/master